apps/dashboard/src/locales/en.json

```diff
@@ -266,7 +269,10 @@
     "payoutDenied": "Successfully denied payout.",
     "pointOfSaleUpdated": "Successfully updated point of sale.",
     "pointOfSaleDeleted": "Successfully deleted point of sale.",
-    "pointOfSaleCreated": "Successfully created point of sale."
+    "pointOfSaleCreated": "Successfully created point of sale.",
+    "waiveFinesSuccess": "Successfully waived fines.",
+    "waiveFinesRejected": "Canceled waiving fines.",
```

Comment:
I would argue that these are not part of a product container operation.

Reply:
Will fix